CHANGELOG.md

```diff
@@ -11,7 +11,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ### Fixed
 - Empty variant activity panel
 - STRs variants popover
-- Split correctly ClinVar multiple significance terms for a variant 
+- Split correctly ClinVar multiple significance terms for a variant
+- Editing the last comment instead of the selected one 
```

Comment:
Just semantics, but I would have written it from the perspective of what it does now, rather than what was wrong before. 🤷‍♂️

Reply:
I was doubtful myself, I'll change it!